### PR TITLE
#4 Estimate Project Size

### PR DESCRIPTION
**Pull request by super132**
For task #4, provide estimate for the project.

### CODE REVIEW

**Comment by super132** [comment]
@dmarkov please review this PR. Thanks.

**Comment by dmarkov** [comment]
@super132 Thanks for the pull request, let me find a reviewer..

**Comment by dmarkov** [comment]
@longtimeago this pull request is for you, please review

**Comment by dmarkov** [comment]
> @dmarkov please review this PR. Thanks.

@super132 thanks, I'll take a look

File: est/jcabi-simpledb.est [hunk 1]
```diff
@@ -0,0 +1,17 @@
+date: 28-05-2015
+author: Jason Wong
+method: champions.pert
+scope:
+  1: jcabi-simpledb project scaffolding
+  2: Implementation of simpledb AWS SDK wrapper
+  3: Achieve 80% test coverage
+  4: CI integration
```
Comment (longtimeago) [inline est/jcabi-simpledb.est]: @super132 what do you mean by CI integration?
We already have travis and rultor configs to enable CI

**Comment by longtimeago** [comment]
@super132 one small question above

File: est/jcabi-simpledb.est (hunk 1 above)
Comment (super132) [inline est/jcabi-simpledb.est]: @longtimeago Travis and Rultor configs are meant to be CI integration. I think this estimate should be thinking from ground up, which means back to the times when this project is newly built, what tasks do we need to do. That is why I put CI integration here as a task to make out the rultor and travis configs. It is not a major task however as you can see in the estimate file. Hope it makes sense. Thanks.

**Comment by super132** [comment]
@longtimeago I replied to your comment. Thanks.

File: est/jcabi-simpledb.est (hunk 1 above)
Comment (longtimeago) [inline est/jcabi-simpledb.est]: @super132 IMHO, this estimation should reflect current state

Comment (longtimeago) [inline est/jcabi-simpledb.est]: @yegor256 what do you think?

Comment (yegor256) [inline est/jcabi-simpledb.est]: @super132 @longtimeago we should always estimate "from scratch", imagining how much time would it take to develop everything from a fresh start

Comment (longtimeago) [inline est/jcabi-simpledb.est]: @yegor256 OK ... I didn't know that. Thank you!

**Comment by longtimeago** [comment]
@rultor merge

**Comment by super132** [comment]
@yegor256 , would you please review this PR? Thanks.

**Comment by yegor256** [comment]
@rultor merge it

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by dmarkov** [comment]
@longtimeago Many thanks! **17 mins** were added to your account in Transaction ID `58584554` (task took 119 hours and 41 mins)

review comments (c=2) added as a bonus

+17 added to your rating, at the moment it is: [+3886](http://www.netbout.com/b/35024?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**